frontend/src/v5/ui/components/shared/modals/modals.styles.ts

```diff
@@ -57,3 +56,15 @@ export const Status = styled(Typography).attrs({
 	margin-bottom: 0;
 	color: ${({ theme }) => theme.palette.base.main};
 `;
+
+export const CircledIcon = styled.div`
+	height: 28px;
+	padding: 33px;
+	border: 2px solid ${({ colour, theme }) => (colour || theme.palette.primary.main)};
+	border-radius: 50%;
+	svg {
+		height: 100%;
+		width: auto;
+		path { fill: ${({ colour, theme }) => (colour || theme.palette.primary.main)} }
+	}
+`;
```

Comment:
Should this be here? Can it be its own control?
Also lets try not to use references to svg and fill if we can, lets change the svg code to use currentColor and use css color property to style the colour.